docs/getting_started/your_first_application.md

```diff
@@ -55,4 +55,7 @@ int main()
     app.port(18080).multithreaded().run();
 }
 ```
-After building and running your .cpp file, you should be able to access your endpoint at [http://localhost:18080](http://localhost:18080). Opening this URL in your browser will show a white screen with "Hello world" typed on it.
+
+For the sake of simplicity, we suggest using `crow_all.h` (by placing it in the same place as your `main.cpp`) and running the command `g++ main.cpp -lpthread -O2 -o first` (use `clang` instead of `g++` if you prefer clang, or Visual Studio if you're on windows).
```

Comment:
I am still unsure about teaching people C++ and its compilation by referring to compilation commands...
For windows users it's a different story. For CMake users it's a different story.

In my opinion the concept of including a header and generating a binary should be known or researched themselves based on the build system they are using.

Reply:
We can have a tabbed section with tutorials for compiling using linux commands, or Visual Studio.

I'm not 100% sure someone will come to the getting started section having knowledge of what CMake is. Since the assumption here is that they're switching from something like nodeJS or Python Flask. And we just want them to have a functioning executable.

Reply:
I guess if language knowledge isn't assumed, this is the way then. 👍